docs/source/developer_documentation/reports.md

```diff
@@ -0,0 +1,79 @@
+# Technical Report Documentation
+
+## Creating reports
+### Location of the report code
+The code of the reports is located inside the `rocky` folder.
+
+### Steps to create a new report
+Make sure rocky is running so you can see the changes you are about to make.
+
+1.	Navigate to the `report_types` folder that contains all the different report types.
+2.	Create a new folder with a descriptive name ending with `_report`. Alternatively, you can duplicate an existing folder, modify the names, and clear its contents. (If you’ve done this, continue to step 3.)
```

Comment:
report naming ending with _report not needed per se

Reply:
All existing reports end with “_reports.” Wouldn't it be better for consistency and readability to write it that way?